pkg/types/field.go

```diff
@@ -98,7 +99,7 @@ func NewSensitiveField(g *Builder, cfg *config.Resource, r *resource, sch *schem
 
 	f.TFTag = "-"
 	switch f.FieldType.String() {
-	case "string", "*string":
+	case "string", "*string", "float64", "*float64":
```

Comment:
Have you tested this change manually and confirmed it works? Here is where we take the value from secret and then write to the TF file but it converts it to `string` before doing so. So, I think we may end up having a `"231231241"` as value instead of `231231241` in the `main.tf.json`, which Terraform may accept but better check if that's the case. Otherwise, we need to find a way to discover the type and convert it to that type before calling `setSensitiveParametersWithPaved` function. cc @turkenh

Reply:
I have only tried upgrading [terraform-provider-datadog](https://github.com/DataDog/terraform-provider-datadog) to version v3.14 which seems to work. But did not try creating a resource with it that uses float64 for sensitive fields check. Will test it out and get back